server/models/Availability.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require("mongoose");
+const Schedule = require("./Schedule");
+
+const availabilitySchema = new mongoose.Schema({
+  schedules: [Schedule]
+});
```

Comment:
So, let's remove this schema and just use the Schedule schema directly. On the pet sitter model we'll just have the `availability` property be a list of schedules.